server/src/main/java/com/objectcomputing/checkins/services/guild/member/GuildMemberController.java

```diff
@@ -26,11 +26,15 @@
 @Secured(SecurityRule.IS_AUTHENTICATED)
 @Produces(MediaType.APPLICATION_JSON)
 @Tag(name = "guild-member")
+@Singleton
```

Comment:
Anything annotated with @Controller also can be injected as a Singleton, so this line is unnecessary.

Reply:
so, if I am understanding this correctly, @Controller sort of implies (or maybe encompasses) @Singleton and thus makes it unnecessary?

Reply:
Correct. If you look at the actual class `io.micronaut.http.annotation.Controller` you'll see the line `@DefaultScope(Singleton.class)` which explicitly gives this annotation the scope of a Singleton.

Reply:
done

Reply:
done